### DISCUSSION TITLE
compile the source code of  milvus-v2.0.0 is failed,how do solve it ?please

### DISCUSSION THREAD

**Discussion by weekstudy** [milvus-io/milvus#15758] [Q&A and General discussion] [answered]
![image](https://user-images.githubusercontent.com/39413303/155687791-09903efb-6079-495e-b936-1e105c259009.png)

platform:Ubuntun18.04
system:linux

**Comment by yhmo** [comment, marked as answer]
Try this GO proxy:
`export GOPROXY=http://goproxy.cn`

**Reply by weekstudy** [reply]
> Try this GO proxy: `export GOPROXY=http://goproxy.cn`

ok
